THIRD-PARTY-NOTICES

```diff
@@ -21,10 +21,14 @@ This project includes:
   AutoService under Apache 2.0
   AutoService Processor under Apache 2.0
   Bean Validation API under The Apache Software License, Version 2.0
+  Bouncy Castle PKIX, CMS, EAC, TSP, PKCS, OCSP, CMP, and CRMF APIs under Bouncy Castle Licence
+  Bouncy Castle Provider under Bouncy Castle Licence
+  Bouncy Castle S/MIME API under Bouncy Castle Licence
   commonmark-java core under BSD 2-Clause License
   Commons CLI under The Apache Software License, Version 2.0
   Commons IO under The Apache Software License, Version 2.0
   FindBugs-jsr305 under The Apache Software License, Version 2.0
+  Gson under Apache 2.0
```

Comment:
Use the Jackson dependency, as the SDK already depends on that library

Reply:
sure